docs/ISSUE_TRIAGE.md

```diff
@@ -0,0 +1,288 @@
+
+In many cases, the issue author or the community is most suitable to contribute changes since they're experts in their domain. Also, it's very common for someone to seek help from the community on a problem.
+
+1. Kindly and politely add a comment which will notify the users subscribed to the issue of any updates.
+   - Explain that the issue would be nice to get resolved, but it isn't prioritized to be worked on by maintainers for the foreseeable future.
+   - If possible or applicable, try to help contributors get started by adding pointers to relevant source code or files as well as adding any helpful input on how to resolve the issue.
+2. Label the issue with `help wanted`.
+3. If applicable, label the issue with `beginner friendly` to denote that the issue is suitable for a beginner to work on.
+
+## Investigation of issues
+
+When an issue has all basic information provided, but the reported problem cannot be reproduced at a first glance, label the issue `triage/needs-information` and post a comment explaining why this label is being applied. Depending on the perceived severity and/or number of upvotes, the investigation will either be delegated to another maintainer for further investigation or put on hold until someone else (maintainer or contributor) picks it up and eventually starts investigating it.
+
+
+Even if you don't have the time or the knowledge to investigate an issue we highly recommend that you upvote the issue if you happen to have the same problem. If you have further details that may help with investigating the issue please provide as much information as possible.
+
+## External pull requests
+
+Part of issue triage should also be triaging of external PRs. The main goal should be to make sure PRs from external contributors have an owner/reviewer and are not forgotten.
+
+1. Check new external PRs which do not have a reviewer.
+1. Check if there is a link to an existing issue.
+1. If not and you know which issue it is solving, add the link yourself, otherwise ask the author to link the issue or create one.
+1. Assign a reviewer based on who was handling the linked issue or what code or feature the PR touches (look at who was the last to make changes there if all else fails).
+
+## GitHub issue management workflow
+
+The following section describes the triage workflow for new GitGHub issues that get created.
+
+### GitHub Issue: Bug
+
+This workflow starts off with a GitHub issue of type bug being created.
+
+1. Collaborator or maintainer creates a GitHub bug using the appropriate GitHub issue template
+2. By default a bug will be created with the `type/bug` and `needs-triage` labels
+
+The following flowchart defines the workflow,
+
+
+```                                                                                                                                                     
```

Comment:
This should just be a PNG that is attached in the doc. Not sure why it needs to be ASCII art. But it's ok for now.

Reply:
I'm resolving this conversation because Jeff has approved the PR.